docs/configuration.md

```diff
@@ -949,6 +953,40 @@ Microsoft Teams notifications are sent via the [Incoming Webhooks](https://learn
 [ http_config: <http_config> | default = global.http_config ]
 ```
 
+### `<msteamsv2_config>`
+
+Microsoft Teams v2 notifications using the new message format with adaptive cards as required by [Workflow](https://support.microsoft.com/en-us/office/creating-a-workflow-from-a-channel-in-teams-242eb8f2-f328-45be-b81f-9817b51a5f0e).
+
+#### How to create a workflow
+
+1. Select the workflow template, `Post to a channel when a webhook request is received`
```

Comment:
I'm hesitant to put these steps in the Alertmanager docs as these might change over time. Perhaps we should just link to their docs instead?

Reply:
Done in: https://github.com/prometheus/alertmanager/pull/4024/commits/b16ceaa2689e23ac5a470f02bea87a8fdb7a2a06